sbomify/apps/core/schemas.py

```diff
@@ -271,6 +273,8 @@ class ComponentUpdateSchema(BaseModel):
     name: str = Field(..., max_length=255, min_length=1)
     component_type: ComponentType = ComponentType.SBOM
     is_public: bool = False
+    # is_global defaults to None so existing clients that omit it do not overwrite scope
```

Comment:
Set it to `False` by default instead.

Reply:
We keep `is_global` as `None` in the API schema so if a client doesn’t send it, we leave the component’s scope unchanged. 
If we defaulted it to False, forgetting to include it in an update would silently make a global component non-global. The database default of False only applies when creating new records.